### PR TITLE
[ST] Update test-clients to 0.6.0

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Enhancement 

### Description

This PR updates version of test-clients to 0.6.0 after latest release.
It also removes the "old" Admin client from the classes - instead, it uses the new Admin-client CLI.
For the CLI tool, I added new templates class, so new static Pod can be created and we can execute all commands from there.
I had to change two tests, which were affected by the change from the old to new Admin-client.

This should also unblock task related to UTO (https://github.com/strimzi/strimzi-kafka-operator/issues/8864)

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/main/java/io/strimzi/systemtest/Environment.java [hunk 1]
```diff
@@ -202,12 +202,9 @@ public class Environment {
 
     // variables for kafka client app images
     private static final String TEST_CLIENTS_VERSION = getOrDefault(TEST_CLIENTS_VERSION_ENV, TEST_CLIENTS_VERSION_DEFAULT);
-    private static final String TEST_CLIENTS_IMAGE_DEFAULT = STRIMZI_REGISTRY_DEFAULT + "/" + TEST_CLIENTS_ORG_DEFAULT + "/test-clients:" + TEST_CLIENTS_VERSION + "-kafka-" + CLIENTS_KAFKA_VERSION;
-    // Admin client is not part of the test-clients, but the changes need to be made in ThrottlingQuotaST to remove this image
-    private static final String TEST_ADMIN_IMAGE_DEFAULT = STRIMZI_REGISTRY_DEFAULT + "/" + TEST_CLIENTS_ORG_DEFAULT + "/test-client-kafka-admin:0.4.2-kafka-3.3.1";
+//    private static final String TEST_CLIENTS_IMAGE_DEFAULT = STRIMZI_REGISTRY_DEFAULT + "/" + TEST_CLIENTS_ORG_DEFAULT + "/test-clients:" + TEST_CLIENTS_VERSION + "-kafka-" + CLIENTS_KAFKA_VERSION;
+    private static final String TEST_CLIENTS_IMAGE_DEFAULT = "quay.io/lkral/test-clients:latest";
```
Comment (see-quick) [inline systemtest/src/main/java/io/strimzi/systemtest/Environment.java:206]: I assume this one will be changed :)

Comment (im-konge) [reply systemtest/src/main/java/io/strimzi/systemtest/Environment.java:206]: Yeah, that's only for testing purposes.

**Review by see-quick — APPROVED** [review]

File: systemtest/src/main/java/io/strimzi/systemtest/Constants.java [hunk 2]
```diff
@@ -97,6 +98,8 @@ public interface Constants {
      */
     String KAFKA_CLIENTS_LABEL_KEY = "user-test-app";
     String KAFKA_ADMIN_CLIENT_LABEL_KEY = "user-test-admin-app";
+    String ADMIN_CLIENT_NAME = "admin-client";
+
```
Comment (jankalinic) [inline systemtest/src/main/java/io/strimzi/systemtest/Constants.java:102]: maybe space

**Review by jankalinic — APPROVED** [review]

**Pull request merged**